source/precalculus/source/08-TE/01.ptx

```diff
@@ -12,6 +12,1187 @@
+          </p>
+        </statement>
+        <answer>
+          <p>
+            Students should label the sides so that the following sides should be labeled as:
+            <ul>
+              <li>
+                <p>
+                  <m>a</m> is the adjacent side of <m>\theta</m>
+                </p>
+              </li>
+              <li>
+                <p>
+                  <m>b</m> is the opposite side of <m>\theta</m>
+                </p>
+              </li>
+              <li>
+                <p>
+                  <m>c</m> is the hypotenuse
+                </p>
+              </li>
+            </ul>
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Replace <m>a</m>, <m>b</m>, and <m>c</m> in the equation we got in part (c) with how you labeled the sides in part (d).
+          </p>
+        </statement>
+        <answer>
+          <p>
+            Replacing the <m>a</m>, <m>b</m>, and <m>c</m> in the equation <me>1+ \left( \frac{b}{a} \right)^2= \left( \frac{c}{a} \right)^2</me> with the labels from part (d) gives <me>1+ \left( \frac{\text{opposite}}{\text{adjacent}} \right)^2=\left( \frac{\text{hypotenuse}}{\text{adjacent}} \right)^2</me>
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Recall the following trigonometric functions:
+            <ul cols="2"> 
+              <li><m>\sin\theta=\frac{\text{opposite}}{\text{hypotenuse}}
+              </m></li>
+              <li><m>
+                \csc\theta=\frac{\text{hypotenuse}}{\text{opposite}}
+              </m></li>
+              <li><m>
+                \cos\theta=\frac{\text{adjacent}}{\text{hypotenuse}}
+              </m></li>
+              <li><m>
+                \sec\theta=\frac{\text{hypotenuse}}{\text{adjacent}}
+              </m></li>
+              <li><m>
+                \tan\theta=\frac{\text{opposite}}{\text{adjacent}}
+              </m></li>
+              <li><m>
+                \cot\theta=\frac{\text{adjacent}}{\text{opposite}}
+              </m></li>
+            </ul>
+            Using these trigonometric ratios, how can we rewrite the equation in part (e)?
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m>1 + \tan^2\theta = \sec^2\theta</m>
+          </p>
+        </answer>
+      </task>
+    </activity>
+
+    <remark>
+      In <xref ref="deriving-pythagorean-identity-2"/>, we derived the second Pythagorean Identity: <m>\sec^2\theta-\tan^2\theta=1</m>. Note that we can rewrite what we got in the previous activity <m> \left( 1 + \tan^2\theta = \sec^2\theta \right) </m> to get to this form. 
+    </remark> 
+
+    <activity xml:id="deriving-pythagorean-identity-3">
```

Comment:
Is it worth the time to go through all of this again (or a third time), or would it be easier to just tell them to take $\sin^2(x)+\cos^2(x)=1$ and divide it by $\sin^2(x)$, etc.?